src/dat.c

```diff
@@ -228,7 +229,7 @@ int eraDat(int iy, int im, int id, double fd, double *deltat )
    if (i < 0) return -5;
 
 /* Get the Delta(AT). */
-   da = changes[(i < 0) ? 0 : i].delat;
+   da = changes[i].delat;
```

Comment:
@timj: We only need one of the fixes here.  The return if i is negative is the better one (but should still probably be upstream in SOFA if it isn't -- don't have time to check right now).